### ISSUE TITLE
Better error message

### ISSUE THREAD

**Issue by davistan** [Surnet/swagger-jsdoc#75]
Hi, currently there will be error from js-yaml when the yaml is invalid.

It will be better to catch error and then show the comment that is causing the issue and filename / line number for easier debugging.

I believe the try catch should be in here - https://github.com/Surnet/swagger-jsdoc/blob/master/lib/index.js#L60

Thanks in advance.

**Comment by kalinchernev** [comment]
Hi @davistan examples will be appreciated ;) 
Just not to spend time assuming things :)

**Comment by davistan** [comment]
Here's the example to make it easier to find the root cause.

```js
      if (tag.title === 'swagger') {
        try {
          swaggerJsDocComments.push(jsYaml.safeLoad(tag.description))
        }
        catch (error) {
          console.error(tag.description)
          throw error
        }
      }
```

**Comment by kalinchernev** [comment]
Hi @davistan I meant example code you use when the error is raised.

**Comment by MartinMuzatko** [comment]
It would be nice to know file, line and column in which the error was thrown. Right now, only line and column of the jsdoc itself is thrown. Not helpful at all if you have multiple definitions

**Comment by kalinchernev** [comment]
Solved by #96
